qubesbuilder/plugins/chroot_deb/__init__.py

```diff
@@ -132,16 +166,40 @@ def run(self, stage: str):
                 )
             ]
             cmd = [
-                f"sed -i '#/tmp/qubes-deb#d' {executor.get_plugins_dir()}/chroot_deb/pbuilder/pbuilderrc",
-                f"mkdir -p {executor.get_cache_dir()}/aptcache",
-            ]
-            pbuilder_cmd = [
-                f"sudo -E pbuilder execute --distribution {self.dist.name}",
-                f"--configfile {executor.get_plugins_dir()}/chroot_deb/pbuilder/pbuilderrc",
-                f"--bindmounts {executor.get_cache_dir()}/aptcache:/tmp/aptcache",
-                f"-- {executor.get_plugins_dir()}/chroot_deb/scripts/apt-download-packages {' '.join(additional_packages)}",
+                quote_args(
+                    "sed",
+                    "-i",
+                    "--",
+                    "#/tmp/eubes-deb#d",
```

Comment:
typo; and also, this exact typo makes pbuilder require "repository" dir, which otherwise wouldn't be needed.